pkg/bindings/containers/containers.go

```diff
@@ -333,3 +339,125 @@ func ContainerInit(ctx context.Context, nameOrID string) error {
 	}
 	return response.Process(nil)
 }
+
+// Attach attaches to a running container
+func Attach(ctx context.Context, nameOrId string, detachKeys *string, logs, stream *bool, stdin, stdout, stderr chan string) error {
```

Comment:
Why are we `chan string` for the standard streams? I would think that io.File so we can directly pass in the stdin/out/err FDs would be a lot better

Reply:
(More accurately, `io.Reader` for stdin, `io.Writer` or `WriteCloser` for stdout/err

Reply:
I went with io.Writer.